docs/source/how_to_guides/inference/how_to_do_bootstrap_inference.ipynb

```diff
@@ -1,16 +1,5 @@
 {
```

Comment:
<p>"than" instead of "from"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/estimagic/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>